src/status_im/ui/components/text_input_with_label/view.cljs

```diff
@@ -2,35 +2,35 @@
   (:require [reagent.core :as r]
             [status-im.ui.components.animation :as animation]
             [status-im.ui.components.text-input-with-label.animation :refer [animate-label
-                                                                          text-input-on-focus
-                                                                          text-input-on-blur]]
+                                                                             text-input-on-focus
+                                                                             text-input-on-blur]]
             [status-im.ui.components.react :refer [view
-                                                text
-                                                animated-text
-                                                animated-view
-                                                text-input]]
-            [status-im.ui.components.text-input-with-label.styles :as st]
-            [clojure.string :as str]))
+                                                   text
+                                                   animated-text
```

Comment:
How about using `as`?

Reply:
I only fixed the bug, of course I can also change everything in ns to be according to our guidelines :)

Reply:
In this case I think it makes sense to take this opportunity for a small cleanup!